x-pack/elastic-agent/control.proto

```diff
@@ -4,10 +4,10 @@
 
 syntax = "proto3";
 
-package proto;
+package cproto;
 
 option cc_enable_arenas = true;
-option go_package = "pkg/agent/control/proto;proto";
+option go_package = "pkg/agent/control/cproto;cproto";
```

Comment:
Can you still upgrade 7.17.x->7.17.x with this change?

In the 8.x agent we maintain two copies of the file but I can't remember if the concern there applies here as well. https://github.com/elastic/elastic-agent/blob/fd0d1bb164dd5dbd0f64c7a2d0e727cf7b13e777/control_v1.proto#L7-L11